src/components/mobile/MobileLayout.js

```diff
@@ -39,21 +40,29 @@ const MobileLayout = () => {
       <TabPanels>
         <TopBarSwitch />
         <Switch>
-          <Route path="/entry/new/details">
+          <Route path={['/map/entry/new/details']}>
             <LocationForm />
           </Route>
-          <Route path={['/map', '/list', '/settings', '/entry/new']}>
-            {tabPanels}
-          </Route>
-          <Route path="/entry/:id">
+          <Route path="/list/entry/:id">
             <EntryTabs />
           </Route>
+          <Route path={['/map', '/list', '/settings', '/map/entry/new']}>
+            <Switch>
+              <Route path={'/map/entry/new'} />
```

Comment:
Is this empty route and switch necessary if the parent route is already in a switch?